### DISCUSSION TITLE
[#958931] Basic InfoBar popup

### DISCUSSION THREAD

**Discussion by conmaster2112** [Mojang/minecraft-editor#100] [Feedback]
We should be able to easily inform the user of our extension when something happens or issue occurs.
Yes we can use chat output, but it weirdly override the UI, me suggestion is 4 basic info Bars.
 - Info
 - Warn
 - Success
 - Error
 
 Here is example from WinUI assets.
 
![image](https://user-images.githubusercontent.com/64863757/227780981-772fd637-6514-4b6c-bc29-e4d31b5f7171.png)
![image](https://user-images.githubusercontent.com/64863757/227780995-dddbd165-e676-4a31-a4d7-55524eca2c55.png)
![image](https://user-images.githubusercontent.com/64863757/227781010-cf574d0e-8efc-4dae-a9b3-72b99bfbaa47.png)
![image](https://user-images.githubusercontent.com/64863757/227781023-4a72bd2e-c92c-4611-a550-c2c0ca4150a8.png)

And Style for more informations, content is on second line.
![image](https://user-images.githubusercontent.com/64863757/227781154-29085de7-0a7a-4d7d-aa2d-dbee28cfd7df.png)

Its just suggestion but could be useful in some cases.
Just little popup in bottom left corner i guess?

**Comment by conmaster2112** [comment]
Here is well known VSCode one,
![image](https://user-images.githubusercontent.com/64863757/227781946-52f2bf89-07e6-45f7-a169-ccf07f58d940.png)

**Comment by minecraftchloe** [comment]
Love this idea a lot! Thank you!

**Comment by minecraftchloe** [comment]
Released a log panel -- Ctrl+H! We plan to expand on this significantly, but it's a start :)
